src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -1230,6 +1244,18 @@ public CaseDataBuilder atStateRespondentRespondToClaimFastTrack(RespondentRespon
         return this;
     }
 
+    public CaseDataBuilder atStateRespondentRespondToClaimPartAdmitFastTrack(
```

Comment:
Please can you make the name consistent with the other added method wrt PartAdmit and FastTrack